src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -56,4 +65,27 @@ public ItemUseHandler getItemHandler() {
         };
     }
 
+    private EntityInteractHandler onRightClickEntity() {
+        return (e, item, hand) -> {
+            if ((e.getRightClicked() instanceof Sheep)) {
```

Comment:
Redundant brackets :eyes:
```suggestion
            if (e.getRightClicked() instanceof Sheep) {
```